app/pages/shared_documents/shared_documents_delete_confirm.php

```diff
@@ -0,0 +1,39 @@
+<?php
+restrict_access([Permission::ROOT]);
+
+$form = new Validator(action: "confirm-delete");
+
+$doc_id = get_route_param("doc_id");
+$shared_doc = SharedFile::get($doc_id);
+if (!$shared_doc) {
+    $form->set_error("Le document numéro $doc_id n'existe pas");
+}
+if ($form->valid()) {
+    if (unlink(app_path() . "/uploads/" . $shared_doc->path)) {
+        logger()->info("File {file_path} deleted by user {currentUserLogin}", ['file_path' => $shared_doc->path, 'currentUserLogin' => User::getCurrent()->login]);
+        em()->remove($shared_doc);
+        em()->flush();
+        Toast::error("Document supprimé");
+        redirect("/documents");
+    } else {
+        $form->set_error("Impossible de supprimer le fichier");
+        logger()->error("File {file_path} not deleted by user {currentUserLogin}", ['file_path' => $shared_doc->path, 'currentUserLogin' => User::getCurrent()->login]);
```

Comment:
I would respect PSR for log formatting here
```suggestion
        logger()->error("File {filePath} not deleted by user {currentUserLogin}", ['filePath' => $shared_doc->path, 'currentUserLogin' => User::getCurrent()->login]);
```

Reply:
✅